### ISSUE TITLE
How to create external source maps?

### ISSUE THREAD

**Issue by teehemkay** [arian/partition-bundle#12]
Subject lines pretty says it all...

Any idea how to proceed to create an external source map for each of the bundles generated by partition-bundle?

Thanks!

**Comment by arian** [comment]
You could have a look at https://github.com/thlorenz/exorcist

https://github.com/substack/node-browserify#external-source-maps
https://github.com/thlorenz/exorcist/wiki/Recipes#gulp

**Comment by teehemkay** [comment]
Thanks for the quick answer.

I should have mentioned that I had already tried to use exorcist exactly as suggested in the browserify documentation at  https://github.com/substack/node-browserify#external-source-maps

But it does not seem to work, I get the following message from exorcist:

```
The code that you piped into exorcist contains no source map!
Therefore it was piped through as is and no external map file generated.
```

I have verified that the bundles generated by partition-bundle **do** include source maps though. 

But in any case I'm not sure how it would work anyway because my partition-bundle config specifies that it should produce **four (4)** different bundles and exorcists only allows for the specification of **one** source map.

Am I missing something obvious here?

So my question is really about how to generate a source map file for _each_ of the bundles produced by partition-bundle.

Hope this clarifies my issue.
